bitnami/postgresql/templates/statefulset.yaml

```diff
@@ -76,6 +76,11 @@ spec:
             - -cx
             - |
               {{- if .Values.persistence.enabled }}
+              {{- if eq ( toString ( .Values.volumePermissions.securityContext.runAsUser )) "auto" }}
+              chown -R `id -u`:`id -G | cut -d " " -f2` {{ .Values.persistence.mountPath }}
```

Comment:
I think we should prevent from performing a recursive operation here. This will affect the entire directory, invalidating the following lines.

We should only give permissions to the `{{ .Values.persistence.mountPath }}` but not to all the sub-directories 😁